tests/installation/addon_products_sle.pm

```diff
@@ -47,26 +48,25 @@ sub run() {
             else {
                 assert_screen "addon-license-$addon";
             }
-            sleep 2;
+            wait_still_screen, 2;
             send_key 'alt-a';                                                   # yes, agree
-            sleep 2;
+            wait_still_screen, 2;
```

Comment:
appears a lot in logfiles, e.g. https://openqa.suse.de/tests/462375/file/autoinst-log.txt

@mnowaksuse please fix